contrib/test_jenkins.sh

```diff
@@ -612,7 +612,9 @@ build_jucx() {
 #
 build_armclang() {
 	echo 1..1 > build_armclang.tap
-	if module_load arm-compiler/latest
+	armclang_test_file=$(mktemp ./XXXXXX).c
```

Comment:
minor: why can't we use just random name? without customization by XXX

Reply:
isn't this what mktemp is doing?

Reply:
by default mktemp creates file in /tmp, we can't use /tmp because it may be cleaned at any moment & test will fail.
this command creates temp file in current directory

Reply:
ok